profile.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+require_once 'profileFunctions.php';
+
+$db = new PDO('mysql:host=127.0.0.1;dbname=portfolio-kevin', 'root');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Profile</title>
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" type="text/css" href="profile.css">
+    <link href="https://fonts.googleapis.com/css?family=Josefin+Sans" rel="stylesheet">
+</head>
+<body>
+    <div class="container">
+        <h1>Profile</h1>
+        <div class="col-4">
+            <img class="profile-image" src="images/profile-photo.png">
+        </div>
+        <div class="col-8">
+            <p>
+                <?php
+                $textArray = getProfileText($db);
+                echo outputProfileText($textArray);
+                ?>
+            </p>
+            <p class="contact">Email:
+                <?php
+                $emailArray = getEmail($db);
```

Comment:
do your logic up the top of the file and just have your output echo line amongst your html